frontend/libs/studio-feedback-form/src/config/FeedbackFormImpl.tsx

```
@@ -2,7 +2,8 @@
 import type { ButtonTexts, QuestionConfig } from '../types/QuestionsProps';
 import { FeedbackFormContextProvider } from '../contexts/FeedbackFormContext';
 import { FeedbackForm } from '../FeedbackForm/FeedbackForm';
-import type { AnswerType } from '../types/AnswerType';
+import { FeedbackFormConfig } from '../types/FeedbackFormConfig';
```

ESLint

[error] All imports in the declaration are only used as types. Use `import type`.

(@typescript-eslint/consistent-type-imports)